src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] arrfigure = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < 3; i++) {
+            arrfigure[i] = figureSupplier.getRandomFigure();
+        }
+        for (int i = 3; i < 6; i++) {
+            arrfigure[i] = figureSupplier.getDefaultFigure();
+        }
+        for (Figure figures : arrfigure) {
+            StringBuilder line = new StringBuilder();
+            if (figures.getClass() == Square.class) {
+                line.append("Figure: square, area: ")
+                        .append(String.format("%(.1f", ((Square) figures).getArea()))
+                        .append(" sq.units, side: ")
+                        .append(((Square) figures).getA())
+                        .append(" units, color: ")
+                        .append(figures.getColor());
```

Comment:
we should have the method `public String draw()` in each Figure and return figure info from it
better create an interface with this method and implement it in Figure class